tests/PySys/apt_plugin/apt_sequence_remove_with_version_fail/run.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import os
+import time
+
+sys.path.append("apt_plugin")
+from environment_apt_plugin import AptPlugin
+
+"""
+Validate apt plugin remove a specific versioned module use case
+
+When we remove a package module with version as below
+When that version is not installed
+Remove will fail
+Grep for the specific error message
+
+sudo /etc/tedge/sm-plugins/apt remove rolldice  --module-version 1.111111
+
+"""
+
+
+class AptPluginRemoveWithVersionFails(AptPlugin):
+    def setup(self):
+        super().setup()
+
+       
+        self.version = "1.111111"
+
+        self.package = "rolldice"
+        self.apt_remove(self.package)
```

Comment:
Wouldn't this corrupt the purpose of this test? If we remove the package itself here, then the actual test in `execute` is just testing the removal of a package that doesn't even exist. I thought what we wanted was to test the removal of an **existing package** failing if the wrong module version is provided in the remove command. So we need to validate that the removal failed, as well as the current version of the package remains on the system and not accidentally removed.

Reply:
definitely, this is wrong, fixed.